mix.exs

```diff
@@ -44,16 +43,18 @@ defmodule Franklin.MixProject do
     [
       # Core Phoenix Deps
       {:ecto_sql, "~> 3.6"},
-      {:esbuild, "~> 0.4", runtime: Mix.env() == :dev},
+      {:esbuild, "~> 0.5", runtime: Mix.env() == :dev},
+      {:finch, "~> 0.14"},
```

Comment:
This was a new addition which is easy to miss. Seems like the preferred networking library for the default Swoosh integration. I'm not doing email currently, but I pulled it in to match the generator.